src/ONYXKEYS.js

```diff
@@ -222,6 +222,9 @@ export default {
     // Whether the auth token is valid
     IS_TOKEN_VALID: 'isTokenValid',
 
+    // The color theme setting defined by the user in preferences
+    COLOR_THEME: 'colorTheme',
```

Comment:
if it's as defined by the user preferences, shouldn't we include `system`? Otherwise lets change this comment